falcon/errors.py

```diff
@@ -377,11 +372,9 @@ class HTTPMethodNotAllowed(OptionalRepresentation, HTTPError):
             base articles related to this error (default ``None``).
     """
 
-    def __init__(self, allowed_methods, headers=None, **kwargs):
+    def __init__(self, allowed_methods, title=None, description=None, headers=None, **kwargs):
```

Comment:
We'll need a breaking change news fragment since people may have been passing _headers_ positionally.

Reply:
I'll add it to the existing one regarding the `NoRepresentation`. Also the kw only change

Reply:
btw `towncrier` does not properly render the misc changelog